contents/handbook/company/docs.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Documentation
+sidebar: Handbook
+showTitle: true
+---
+
+## Responsibilities
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
+  - Correcting mistakes reported by users.
+  - Clarifying documentation based on user feedback and support tickets.
+
+- **Content marketing** is responsible for improving the docs. This means:
+  - Polishing draft documentation created by product teams.
+  - Identifying and improving low quality documentation.
+  - Ensuring screenshots and other visual elements are up-to-date.
+  - Shipping supplementary docs and tutorials based on feedback and emerging use cases.
```

Comment:
Im a bit concerned this implies product teams can just ship rubbish in the hopes we'll clean up after them. Can we add a bullet point in the **Product teams** section that says something along the lines of "you can tag the content marketing team to review your PRs to ensure your writing is clear" (obviiously phrased nicer)

Reply:
Its not clear on github but my comment was not refering to line 18 specifically, but rather lines 9-18 collectively

Reply:
Oh, I see.

Looking at this again, I think no wording tweak is going to eliminate laziness / bad faith drafts that pushes the work onto other people. I think, given our culture, it's reasonable to expect people to do a half decent job and, if fail to do so, this should be addressed via feedback.